packages/web/src/components/search/SearchBox.test.js

```diff
@@ -660,3 +405,194 @@ it('should render SearchBox with featured suggestions', () => {
 		.toJSON();
 	expect(elem).toMatchSnapshot();
 });
+
+it('should render SearchBox with FAQ suggestions', () => {
+	const elem = renderer
+		.create(
+			<ReactiveBase app="test" url="https://a03a1cb71321:75b6603d-9456-4a5a-af6b-a487b309eb61@localhost:800">
+				<SearchBox
+					testMode
+					componentId="MockSearchBox"
+					dataField="original_title"
+					mockData={{ hits: FAQ_SUGGESTIONS }}
+					isOpen
+				/>
+			</ReactiveBase>,
+		)
+		.toJSON();
+	expect(elem).toMatchSnapshot();
+});
+
+it('should render SearchBox with Document suggestions', () => {
+	const elem = renderer
+		.create(
+			<ReactiveBase app="test" url="https://a03a1cb71321:75b6603d-9456-4a5a-af6b-a487b309eb61@localhost:800">
+				<SearchBox
+					testMode
+					componentId="MockSearchBox"
+					dataField="original_title"
+					mockData={{ hits: DOCUMENT_SUGGESTIONS }}
+					isOpen
+				/>
+			</ReactiveBase>,
+		)
+		.toJSON();
+	expect(elem).toMatchSnapshot();
+});
+
+it('should render AI response', async () => {
+	const elem = renderer
+		.create(
+			<ReactiveBase app="test" url="https://a03a1cb71321:75b6603d-9456-4a5a-af6b-a487b309eb61@localhost:800">
+				<SearchBox
+					testMode
+					componentId="MockSearchBox"
+					dataField="original_title"
+					mockData={{
+						hits: AI.MOCK_HITS_DATA,
+						AI_RESPONSE: AI.MOCK_AI_RESPONSE,
+						rawData: AI.MOCK_RAW_DATA,
+					}}
+					enableAI
+					isOpen
+				/>
+			</ReactiveBase>,
+		);
+
+	// Wait for all updates to complete
+	await act(async () => {
+		await new Promise(resolve => setTimeout(resolve, 0));
+	});
+
+	expect(elem.toJSON()).toMatchSnapshot();
+});
+
+
+it('should render AI response & askButton & enterButton', async () => {
```

Comment:
https://github.com/appbaseio/reactivesearch/blob/acb9c236a5daf96f77c1b0a86d37587636a21078/packages/web/src/components/search/SearchBox.js#L1061-L1062

We should change the class names for askbutton from enter-btn or enter-button-wrapper to ask-btn and ask-button-wrapper, please update the snapshot post this change

Reply:
What should I do here?